ansible/playbooks/service/nullmailer.yml

```diff
@@ -13,6 +13,12 @@
     - role: debops.nullmailer/env
       tags: [ 'role::nullmailer', 'role::ferm', 'role::tcpwrappers' ]
 
+    - role: debops.secret
+      tags: [ 'role::secret', 'role::nullmailer' ]
+      secret__directories:
+        - '{{ etc_aliases__secret__directories }}'
+        - '{{ postfix__secret__directories }}'
+
```

Comment:
This is not needed in the playbook, having the dependency in `meta/main.yml` of a role makes it a "hard dependency" which means that it will always be included in the run.

Also, you referenced a variable `postfix__secret__directories` which is not in any of the included roles and will result in Ansible erroring out.

Reply:
Apparently I have not drank enough coffee this morning :coffee:

Reply:
@drybjed So I just need to edit the meta information without touching the playbook?

Reply:
@rei-ifesca Yes, exactly. Just modify `meta/main.yml` and you're good.